Sources/MapboxNavigation/Cache.swift

```diff
@@ -27,6 +27,51 @@ public protocol BimodalDataCache: BimodalCache {
     func data(forKey: String?) -> Data?
 }
 
+protocol URLCaching {
+    func store(_ cachedResponse: CachedURLResponse, for request: URLRequest)
+    func response(for request: URLRequest) -> CachedURLResponse?
+    func clearCache()
+    func removeCache(for request: URLRequest)
+}
+
+/**
+ A general purpose URLCache used by `SpriteRepository` implementations.
+ */
+internal class URLDataCache: URLCaching {
+    let diskCacheURL: URL = {
+        let fileManager = FileManager.default
+        let basePath = fileManager.urls(for: .cachesDirectory, in: .userDomainMask).first!
+        let identifier = Bundle.mapboxNavigation.bundleIdentifier!
+        return basePath.appendingPathComponent(identifier).appendingPathComponent("URLDataCache")
+    }()
+    
+    let urlCache: URLCache
+    
+    init() {
+        if #available(iOS 13.0, *) {
+            urlCache = URLCache(memoryCapacity: 5 * 1024 * 1024, diskCapacity: 5 * 1024 * 1024, directory: diskCacheURL)
```

Comment:
We can expose initializer that allows to set `memoryCapacity` and `diskCapacity` as well. These constants can also be extracted into separate constants to prevent duplication.

Reply:
Thanks for the suggestion, the parameter for `memoryCapacity` and `diskCapacity`  are added. I'll add test for the capacity limit.